bridge_adaptivity/module/tests/test_views.py

```diff
@@ -40,13 +40,16 @@ def setUp(self):
         self.test_cg.collections.add(self.collection1)
         self.test_cg.collections.add(self.collection3)
 
-        self.group_post_data = self.add_prefix(self.group_prefix, {
+        self.group_update_data = {
             'name': "CG2",
             'collections': [self.collection1.id, self.collection2.id, self.collection3.id],
             'engine': self.engine.id,
             'owner': self.user.id,
-            'grading_policy_name': 'trials_count'
-        })
+            'grading_policy_name': 'trials_count',
+            'description': 'Some description for a group'
+        }
+
+        self.group_post_data = self.add_prefix(self.group_prefix, self.group_update_data)
```

Comment:
It would be nice if the description is shown in the collection_group list view.

Reply:
sure, I will add it.